internal/resources/policy/operations/helper.go

```diff
@@ -20,13 +20,15 @@ import (
 	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/policy"
 	policykindcustom "github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/policy/kind/custom"
 	policykindimage "github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/policy/kind/image"
+	policykindquota "github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/policy/kind/quota"
```

Comment:
Why have repo path as policy/kind/...? Is the kind actually required?

Reply:
We are naming it as `policykind` as we have refactored operations and the type of policy in separate directories, so describing as kind made sense, let me know if that needs to be changed, but we will have to take it in a separate PR, as all previous policies will also required to be changed accordingly.

Reply:
Kind is not adding much value, it can be removed but at the sometime it's not too long, so for now we can live with this. 
If the import alias gets too big then consider removing it